### ISSUE TITLE
[BUG] invalid syntax

### ISSUE THREAD

**Issue by Vorffeed** [charlesbel/Microsoft-Rewards-Farmer#107]
When I try to start the .py file, I get this error

```
File "C:\Microsoft-Rewards-Farmer-master\ms_rewards_farmer.py", line 26
    def browserSetup(headless_mode: bool = False, user_agent: str = PC_USER_AGENT) -> WebDriver:
                                  ^
SyntaxError: invalid syntax
```

Does anyone know what it is and how to fix it.
because I have followed all the steps

**Comment by Andrew1644563** [comment]
Did you change anything in the code from the original script?

**Comment by Vorffeed** [comment]
I haven't touched it at all, it's the original.

**Comment by Prabh-Saini** [comment]
I know I'm a little late, but if you haven't figured it out yet try doing `py --version`, and report it here

**Comment by Vorffeed** [comment]
Python 3.10.4

**Comment by charlesbel** [comment]
Fixed with v3.0 #188
